german/schwimmen/fluegel_freischneiden/fluegel_freischneiden.md

```diff
@@ -0,0 +1,4 @@
+Dort hinten, an den Klippen aus scharfkantigen Felsen wird sich etwas finden lassen. 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.